### PR TITLE
[DOCS-7850] Alfresco Content Accelerator 3.7 Release

### PR DESCRIPTION
**Pull request by monikamichalska**

### CODE REVIEW

**Review by anxumalo — COMMENTED** [review]
Sending initial feedback - see inline comments

File: content-accelerator/3.7/configure/actions.md
[the remark this answers is not in the document]
Comment (monikamichalska) [reply content-accelerator/3.7/configure/actions.md:1]: Fixed in the next commit (https://github.com/Alfresco/docs-alfresco/pull/1573/commits/1b65820989984cb28af40863056195e7ed8eb62d)

File: content-accelerator/3.7/support/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Supported platforms
+---
+
+The following are the supported platforms for the Alfresco Content Accelerator 3.6
```
[the remark this answers is not in the document]
Comment (monikamichalska) [reply content-accelerator/3.7/support/index.md]: Fixed in the next commit (https://github.com/Alfresco/docs-alfresco/pull/1573/commits/1b65820989984cb28af40863056195e7ed8eb62d)

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**